manifests/kustomize/OWNERS

```diff
@@ -0,0 +1,8 @@
+approvers:
+  - tarilabs
+  - rareddy
+  - Tomcli
+reviewers:
+  - tarilabs
+  - rareddy
+  - Tomcli
```

Comment:
do we _actually_ need this file here?
to me the requirement is covered already here: 

https://github.com/kubeflow/internal-acls/blob/35b98a2763bcfdfc92b87b43a3358b9664398de4/github-orgs/kubeflow/org.yaml#L1036-L1048

@rimolive what are the Manifest wg thoughts on this one, please?
Can we just rely on the file from `kubeflow/internal-acls` ?

Reply:
Manifests repo has multi-level OWNERS files, meaning that depending on what part of the manifests you are contributing with changes, you need ackowledgement of the WG leaders responsible for that part. It applies to all component manifests even when we're working on the sync manifests phase. It is relevant because adding changes to Model Registry need an ack from the WG leaders. It's fine to me if you copy from `kubeflow/internal-acls` but I'll defer to @juliusvonkohout to answer that question.

Reply:
I see, you want this here so it's _also_ copied over to KF/manifest, so it will get applicable there, sorry I didn't consider that 👍 
resolving this thread.